app/register/register.go

```diff
@@ -217,17 +217,17 @@ func (c clusterRegisterUtil) UnRegisterNatsSyncClient() error {
 
 	if err != nil {
 		if response != nil {
-			return errors.New(CreateErrorMsg("UnRegisterNatsSyncClient", "make POST call", natsSyncClientUnregisterURL, response.Status, "", err))
+			return errors.New(CreateErrorMsg("UnRegisterNatsSyncClient", "make POST call", natsSyncClientUnregisterURL, response.Status, "", "", err))
 		}
-		return errors.New(CreateErrorMsg("UnRegisterNatsSyncClient", "make POST call", natsSyncClientUnregisterURL, "", "", err))
+		return errors.New(CreateErrorMsg("UnRegisterNatsSyncClient", "make POST call", natsSyncClientUnregisterURL, "", "", "", err))
 	}
 
 	if response.StatusCode != http.StatusNoContent {
 		bodyBytes, err := io.ReadAll(response.Body)
 		if err != nil {
-			return errors.New(CreateErrorMsg("UnRegisterNatsSyncClient", "read response", natsSyncClientUnregisterURL, response.Status, "", err))
+			return errors.New(CreateErrorMsg("UnRegisterNatsSyncClient", "read response", natsSyncClientUnregisterURL, response.Status, "", "", err))
 		}
-		return errors.New(CreateErrorMsg("UnRegisterNatsSyncClient", "make POST call", natsSyncClientUnregisterURL, response.Status, string(bodyBytes), errors.New("Unexpected unregistration status")))
+		return errors.New(CreateErrorMsg("UnRegisterNatsSyncClient", "make POST call", natsSyncClientUnregisterURL, response.Status, string(bodyBytes), "", errors.New("Unexpected unregistration status")))
```

Comment:
Should errors in go be non-capitalized with no punctuation?

s/"Unexpected unregistration status"/"unexpected unregistration status"